src/main/java/uk/gov/digital/ho/pttg/testrunner/ComponentHeaderChecker.java

```diff
@@ -22,10 +24,25 @@ public boolean checkAllComponentsPresent(List<String> componentTraceHeaders) {
                                                              .flatMap(ComponentHeaderChecker::splitComponents)
                                                              .map(String::trim)
                                                              .collect(Collectors.toSet());
-        return componentsPresent.equals(ALL_EXPECTED_COMPONENTS);
+
+        boolean allPresent = componentsPresent.equals(ALL_EXPECTED_COMPONENTS);
+        logMissingComponents(componentsPresent, allPresent);
+        return allPresent;
     }
 
     private static Stream<? extends String> splitComponents(String header) {
         return Arrays.stream(header.split(","));
     }
+
+    private void logMissingComponents(Set<String> componentsPresent, boolean allPresent) {
+        if (!allPresent) {
```

Comment:
Should we log something out if all components are present?  It's easier to search for a success message in the logs than to search for the absence of an error message.

Reply:
As the Resource logs success/failure it _should_ be possible to work it out easily. But another log won't hurt and will probably make it easier.